tests/features/community_content/content_sharing.feature

```diff
@@ -157,6 +157,86 @@ Feature: Sharing content between collections
+    And the following collection user memberships:
+      | collection | user       | roles       |
+      | Westeros   | John snow  | facilitator |
+      | Essos city | John snow  | member      |
+      | Essos city | Arya Stark | facilitator |
+    When I am logged in as "John Snow"
+    And I click "Keep up to date"
+    Then I should see the contextual link "Share" in the "Iron throne" tile
+    But I should not see the contextual link "Unshare" in the "Iron throne" tile
+
+    When I am logged in as "Arya Stark"
+    And I click "Keep up to date"
+    Then I should see the contextual link "Share" in the "Iron throne" tile
+    And I should not see the contextual link "Unshare" in the "Iron throne" tile
+
+    When I am logged in as "John Snow"
+    # Normally we would share directly from the Keep up to date page. However, a styling issue is preventing the
+    # checkbox to be located.
+    # @see: ISAICP-5245.
+    # And I click "Keep up to date"
+    # Then I should see the contextual link "Share" in the "Iron throne" tile
+    # And I click the contextual link "Share" in the "Iron throne" tile
+    When I go to the content page of the type "<content type>" with the title "Iron throne"
+    And I click "Share"
+    Then a modal should open
+    And the following fields should be present "Essos city"
+    When I check "Essos city"
+    And I press "Share" in the "Modal buttons" region
+    And I wait for AJAX to finish
+    Then I should see the success message "Item was shared in the following collections: Essos city."
+
+    When I am on the homepage
+    And I click "Keep up to date"
+    Then I should see the contextual link "Share" in the "Iron throne" tile
+    # Simple members can still not unshare content from collections.
+    But I should not see the contextual link "Unshare" in the "Iron throne" tile
+
+    When I am logged in as "Arya Stark"
+    And I click "Keep up to date"
+    Then I should see the contextual link "Share" in the "Iron throne" tile
+    # Link vary by user og role since the 2 users up to now have the same permissions outside og.
+    And I should see the contextual link "Unshare" in the "Iron throne" tile
+    When I click the contextual link "Unshare" in the "Iron throne" tile
+    Then a modal should open
+    And the following fields should be present "Essos city"
+
+    When I am logged in as "Jamie Lanister"
+    And I click "Keep up to date"
+    Then I should see the contextual link "Share" in the "Iron throne" tile
+    And I should see the contextual link "Unshare" in the "Iron throne" tile
+    When I click the contextual link "Unshare" in the "Iron throne" tile
+    Then a modal should open
+    And the following fields should be present "Essos city"
+
+    When I am logged in as "John Snow"
+    And I click "Keep up to date"
+    Then I should see the contextual link "Share" in the "Iron throne" tile
+    And I should not see the contextual link "Unshare" in the "Iron throne" tile
```

Comment:
It is checked twice that John Snow doesn't have access to the Unshare button after the content is shared. I guess this makes it _really_ sure that it works, but it is not strictly needed :smile: